dbt/adapters/snowflake/impl.py

```diff
@@ -76,17 +76,18 @@ def open_connection(cls, connection):
 
         try:
             credentials = connection.credentials
+            # Pull all of the optional authentication args for the connector, let connector handle the actual arg validation 
+            auth_args = { auth_key:credentials[auth_key] for auth_key in ['user', 'password', 'authenticator'] if auth_key in credentials}
```

Comment:
this will fail pep8, can you break up this line so that it's shorter than 80 characters?